src/MoviePage/MoviePage.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import '../Error/Error.scss'
+
+const MoviePage = (props) => {
```

Comment:
just now seeing this, disregard comment above. Good deal. Will need to add styling to late night thoughts issue